lib/ansible/modules/dnf.py

```diff
@@ -172,6 +172,13 @@
     type: bool
     default: "yes"
     version_added: "2.7"
+  sslverify:
+    description:
+      - Disables SSL validation of the repository server for this transaction.
+      - This should be set to C(no) if one of the configured repositories is using an untrusted or self-signed certificate.
+    type: bool
+    default: "yes"
+    version_added: "2.12"
```

Comment:
This would have to be `2.13` as `2.12` is released already and we do not backport features. Same below.

Reply:
thanks, I fixed it.